WinApp/audiobookshelf/TrayOnly.cs

```diff
@@ -155,12 +152,12 @@ private void openBrowser(string port)
             Process.Start(psInfo);
         }
 
-
-        private void OuputHandler(object sendingProcess, DataReceivedEventArgs outLine)
-        {
-            // Console.WriteLine(outLine.Data);
-            Debug.WriteLine(outLine.Data);
-        }
+        // Why??
```

Comment:
When testing that the server was running correctly it was useful to see the server logs, this would output the server logs.